pos_show_clock/static/src/js/ChromeWidgets/Clock.js

```diff
@@ -0,0 +1,50 @@
+odoo.define("pos_show_clock.Clock", function (require) {
+    "use strict";
+
+    const PosComponent = require("point_of_sale.PosComponent");
+    const Registries = require("point_of_sale.Registries");
+    const {useState} = owl;
+
+    class Clock extends PosComponent {
+        setup() {
+            super.setup();
+            this.state = useState({current_time: this.time});
+        }
+
+        get now() {
+            return moment(Date.now());
+        }
+
+        get timeFormat() {
+            return "HH:mm";
+        }
+
+        get time() {
+            return this.now.format(this.timeFormat);
+        }
+
+        mounted() {
+            this.intervalRef = setInterval(() => this.updateCurrentTime(), 1000);
+        }
+
+        willUnmount() {
+            if (this.intervalReference) {
+                clearInterval(this.intervalReference);
+            }
+        }
```

Comment:
Looks much better now! Thanks!

I think there's a typo here, thuogh:

On `mounted` the interval reference is stored in `internalRef`.
On `willUnmount`, it's checking and clearing `intervalReference`.

Reply:
You're right, thanks! I just fixed it